src/main/java/com/bean00/Parser.java

```diff
@@ -0,0 +1,47 @@
+package com.bean00;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class Parser {
```

Comment:
I'm thinking to change the name to `HTTPRequestParser` or `RequestParser`. I'm leaning towards `RequestParser` (and will use it for now), because the project is an HTTP Server, so it can be implied that the request is an HTTP request (I also prefer the shorter name `RequestParser`).